app/controllers/handlers/AddExerciseHandler.scala

```diff
@@ -4,12 +4,15 @@ import java.util.Calendar
 
 import controllers.Application._
 import forms.{ExerciseEntriesForm, ExerciseEntryFormInput}
-import models.{ExerciseEntries, ExerciseEntry}
+import models.{ExerciseEntries, ExerciseEntry, Users}
 import play.api.data.{Form, FormError}
 import play.api.mvc.{AnyContent, Request, Result}
 import utils.authentication.UserAuthenticator
+import utils.groupme.Bot
 
 object AddExerciseHandler {
+  lazy val bot = new Bot(sys.env("GROUPME_BOT_ID"), sys.env("GROUPME_URL"))
```

Comment:
I put it here because I wasn't sure how to put it in `Global`. :smile: 

Feel free to make commits on this branch to do that!